bin/gen-schemas.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -xe
 
+set -e
```

Comment:
Why are you now blindly setting this flag everywhere? Maybe not all scripts are ok with that. Did you test them all?

Please be aware that some scripts might invoke failing commands to test something.

Reply:
Sorry, I misunderstood your earlier comment.